src/flamegpu/gpu/CUDAEnsemble.cu

```diff
@@ -310,6 +310,11 @@ int CUDAEnsemble::checkArgs(int argc, const char** argv) {
             config.timing = true;
             continue;
         }
+        // -u/--silence-unknown-args, Silence warning for unknown arguments
+        if (arg.compare("--silence-unknown-args") == 0 || arg.compare("-u") == 0) {
```

Comment:
Does this need a short option? `-u` is not the most memorable for `silence  unknown arguments`, and there is a finite set of short options available (not that we wil ever have 52).

Reply:
I don't see any harm.